devops/shiny_server.conf

```diff
@@ -17,7 +20,7 @@ server {
     directory_index on;
 
     # Disable some network protocols that are causing issues
-    disable_protocols websocket xdr-streaming xhr-streaming iframe-eventsource iframe-htmlfile;
+    # disable_protocols websocket xdr-streaming xhr-streaming iframe-eventsource iframe-htmlfile;
```

Comment:
are these lines still needed?

Reply:
not needed but good to keep since the protocol issue caused a lot of issues in the past. see also #128